app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
I changed the code to reuse existing `save_custom_attribute_attribute_inventory`.
I kept the `additional_attributes` alias same way we do for `labels`, for easier access and management and to differentiate those custom_attributes from others.

@blomquisg PTAL